back/src/main/java/com/cocktailpick/back/cocktail/controller/CocktailController.java

```diff
@@ -37,29 +39,28 @@
 	private final CocktailRecommendService cocktailRecommendService;
 
 	@GetMapping
-	public ResponseEntity<List<CocktailResponse>> findCocktails() {
-		return ResponseEntity.ok(cocktailService.findAllCocktails());
+	public ResponseEntity<List<CocktailResponse>> findCocktails(@CurrentUser User user) {
+		return ResponseEntity.ok(cocktailService.findAllCocktails(user.getFavorites()));
```

Comment:
```suggestion
		return ResponseEntity.ok(cocktailService.findAllCocktails(user);
```

이렇게 하고 서비스 안에서 꺼내는 것이 좋을 것 같습니다. 현재는 컨트롤러가 로직을 가지는 느낌이어서